### DISCUSSION TITLE
Respawn points and Tickets and interpreter

### DISCUSSION THREAD

**Discussion by KDaum50** [Vdauphin/HeartsAndMinds#1396] [Question and answer]
What number do I need to change for the players to only spawn at the base? and how does the tickets work exactly are they player based, or hole team based? want to get the tickets setup right for a large-scale operation with a ton of players, so for example I want each player to have a total of 4 respawns or is it Team based so if I have 40 players and every time one dies it takes away tickets from everyone? How does a player become an interpreter? is there a code I need to put in int. field or something?

**Comment by Vdauphin** [comment]
Hello,
> What number do I need to change for the players to only spawn at the base?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L50-L55

> and how does the tickets work exactly are they player based, or hole team based? want to get the tickets setup right for a large-scale operation with a ton of players, so for example I want each player to have a total of 4 respawns or is it Team based so if I have 40 players and every time one dies it takes away tickets from everyone? 

http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#respawn-tickets
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L74-L79

> How does a player become an interpreter? is there a code I need to put in int. field or something?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/mission.sqm#L786

Cheers

**Reply by KDaum50** [reply]
What number do I need to change for the players to only spawn at the base? 
So, I need to change 4 to 0?

**Reply by KDaum50** [reply]
Is this right
![20220815165501_1](https://user-images.githubusercontent.com/110949312/184725116-1412d0fc-8452-41e2-9226-51a117ff51de.jpg)
![20220815165417_1](https://user-images.githubusercontent.com/110949312/184725138-27c7c0a0-598d-42ba-a2e4-6051a2172d3d.jpg)
![20220815165329_1](https://user-images.githubusercontent.com/110949312/184725149-090126ed-6b95-4a89-af65-69280ae36bcf.jpg)


?

**Reply by Vdauphin** [reply]
> So, I need to change 4 to 0?

0?

> is this right

No, check in the init field of the interpreter

**Reply by KDaum50** [reply]
Ok this mean they not shared right?  $STR_BTC_HAM_RESP_NOTSHARE
No, check in the init field of the interpreter ???? what????

**Reply by KDaum50** [reply]
Change the 4 to 0 
class btc_p_respawn_location { // Respawn locations available:
        title = __EVAL(format ["      %1", localize "STR_BTC_HAM_RESP_LOCATION"]);
        values[]={0,1,2,3,4};
        texts[]={$STR_BTC_HAM_RESP_FOBRALLY, $STR_BTC_HAM_RESP_FOBRALLYHELO, $STR_BTC_HAM_RESP_FOBRALLYHELI, $STR_BTC_HAM_RESP_FOBRALLYHELIVEHI, $STR_BTC_HAM_RESP_FOBRALLYHELIVEHILEADER};
        default = 4;

**Reply by Vdauphin** [reply]
> No, check in the init field of the interpreter ???? what????

Download hearts and minds, go in EDEN editor, double clic on the interpreter, look at the init field
For such basic question you can join the official A3 Discord

**Reply by KDaum50** [reply]
Where is the interpreter? cause the mission i have there's not one

**Reply by Vdauphin** [reply]
> Where is the interpreter?

It is the guy named "interpreter"

**Reply by KDaum50** [reply]
yep not one in there I'll re-download the mission

**Comment by KDaum50** [comment]


**Reply by KDaum50** [reply]
I keep getting this error

**Reply by Vdauphin** [reply]
Join the A3 Discord they will help you to fix your A3
